prow/plugins/override/override.go

```diff
@@ -41,7 +41,7 @@ import (
 const pluginName = "override"
 
 var (
-	overrideRe = regexp.MustCompile(`(?mi)^/override( (.+?)\s*)?$`)
+	overrideRe = regexp.MustCompile(`(?mi)^/override( (.+)\s*)*$`)
```

Comment:
Indeed I'm not convinced by your regex... few questions (without playing with it):

- why did you drop the lazy match `.+?` for  a greedy
- I think you need more parenthesis because the outer capture group repeats now, so it should only match the last iteration

Reply:
* Indeed, with the usage of ``strings.Fields``` I don't need this regex change is needed at all. Reverted
* All tests still passing, so please let me know if there are any edge cases that are missing tests and I can add them in this PR or another

Reply:
In the end I think it can even be simplified into ```(?mi)^/override( (.+))?$```, because we'll strip all whitespaces either way with the ``strings.Fields``